assets/i18n/hi.json

```diff
@@ -0,0 +1,276 @@
+  "I18N_LIBRARY_N_CATEGORIES": "{categoriesCount, plural, =1{1 Category} other{# Categories}}",
+  "I18N_LIBRARY_N_LANGUAGES": "{languagesCount, plural, =1{1 Language} other{# Languages}}",
+  "I18N_LIBRARY_PAGE_TITLE": "एक्सप्लोरेशन पुस्तकालय - Oppia",
+  "I18N_LIBRARY_RATINGS_TOOLTIP": "रेटिंग्स",
+  "I18N_LIBRARY_SEARCH_PLACEHOLDER": "आप किस बारे में उत्सुक हैं ?",
+  "I18N_LIBRARY_SUB_HEADER": "हमारे एक्सप्लोरेशन्स ब्राउज़ कर अपना साहसिक शुरू करे।",
+  "I18N_LIBRARY_UPDATED": "अद्यतन",
+  "I18N_LIBRARY_VIEWS_TOOLTIP": "मत",
+  "I18N_LIBRARY_VIEW_ALL": "सभी को देखें",
+  "I18N_PLAYER_BACK": "वापस",
+  "I18N_PLAYER_CARD_NUMBER_TOOLTIP": "कार्ड #",
+  "I18N_PLAYER_COMMUNITY_EDITABLE_TOOLTIP": "समुदाय संपादन योग्य",
+  "I18N_PLAYER_CONTINUE_BUTTON": "जारी रहना",
+  "I18N_PLAYER_CONTINUE_NEXT_CARD": "अगले कार्ड पे जाये",
+  "I18N_PLAYER_CONTRIBUTORS_TOOLTIP": "योगदानकर्ताओं",
+  "I18N_PLAYER_EDIT_TOOLTIP": "संपादित",
+  "I18N_PLAYER_EMBED_TOOLTIP": "जड़ना",
+  "I18N_PLAYER_FEEDBACK_TOOLTIP": "फीडबैक",
+  "I18N_PLAYER_FORWARD": "आगे",
+  "I18N_PLAYER_INFO_TOOLTIP": "जानकारी",
+  "I18N_PLAYER_LAST_UPDATED_TOOLTIP": "आखरी अपडेट",
+  "I18N_PLAYER_LEAVE_FEEDBACK": "लेखकों के लिए प्रतिक्रिया छोड़े...",
+  "I18N_PLAYER_LOADING": "लोड हो रहा है...",
+  "I18N_PLAYER_NO_OBJECTIVE": "कोई मुद्दा नहीं निर्दिष्ट।",
+  "I18N_PLAYER_NO_TAGS": "कोई टैग्स नहीं जुड़े है।",
+  "I18N_PLAYER_PLUS_TAGS": "<[additionalTagNumber]>+ अधिक टैग",
+  "I18N_PLAYER_PREVIOUS_RESPONSES": "पिछले प्रतिक्रियाएं (<[previousResponses]>)",
+  "I18N_PLAYER_RATE_EXPLORATION": "आपने कुछ नया सीखा ? आप इस एक्सप्लोरशन को कैसे रेटिंग देंगे ?",
+  "I18N_PLAYER_RATINGS_TOOLTIP": "रेटिंग्स",
+  "I18N_PLAYER_RECOMMEND_EXPLORATIONS": "अगले सुझाव:",
+  "I18N_PLAYER_RETURN_TO_COLLECTION": "वापस जाये <[collectionTitle]>",
+  "I18N_PLAYER_RETURN_TO_EDITOR": "संपादक पर वापस जाये",
+  "I18N_PLAYER_RETURN_TO_LIBRARY": "पुस्तकालय पर वापस जाये",
+  "I18N_PLAYER_SHARE_EXPLORATION": "आपने इस एक्सप्लोरेशन का आननद लिया ? अपने दोस्तों के साथ शेयर करे! ",
+  "I18N_PLAYER_STAY_ANONYMOUS": "गुमनाम रहे",
+  "I18N_PLAYER_SUBMIT_BUTTON": "जमा करें",
+  "I18N_PLAYER_TAGS_TOOLTIP": "टैग",
+  "I18N_PLAYER_THANK_FEEDBACK": "प्रतिक्रिया के लिए आपका धन्यवाद!",
+  "I18N_PLAYER_UNRATED": "अनरेटेड",
+  "I18N_PLAYER_VIEWS_TOOLTIP": "मत",
+  "I18N_PREFERENCES_BIO": "जैव",
+  "I18N_PREFERENCES_BIO_EXPLAIN_TEXT": "यह फ़ील्ड वैकल्पिक है। यहाँ कुछ भी लिखा हुआ सार्वजनिक है और पूरा विश्व देख सकता है ।",
+  "I18N_PREFERENCES_BREADCRUMB": "तरजीह",
+  "I18N_PREFERENCES_CANCEL_BUTTON": "रद्द करे",
+  "I18N_PREFERENCES_CHANGE_PICTURE": "प्रोफ़ाइल चित्र बदलें",
+  "I18N_PREFERENCES_EMAIL": "ईमेल",
+  "I18N_PREFERENCES_EMAIL_EXPLAIN": "केवल मध्यस्थों और साइट व्यवस्थापक यह देख सकते हैं ।",
+  "I18N_PREFERENCES_EMAIL_RECEIVE_NEWS": "साइट के बारे में समाचार प्राप्त करे।",
+  "I18N_PREFERENCES_HEADING": "प्राथमिकताएँ",
+  "I18N_PREFERENCES_HEADING_SUBTEXT": "आपके द्वारा की गयी किसी भी प्रकार की परिवर्तन इस पेज पर ऑटो सेव हो जाएगी। ",
```

Comment:
"आपके द्वारा किया गया किसी भी प्रकार का परिवर्तन इस पेज पर ऑटो सेव हो जाएगा।" is better to read and understand. I guess.